docs/content/latest/sample-data/chinook.md

```diff
@@ -38,14 +38,16 @@ To install and use the Chinook sample database, you need to have installed and c
 
 Install the Chinook sample database
 
-### 1. Download the SQL scripts
+### 1. Download the SQL scripts (Optional)
 
-You can download the Chinook database that is compatible with YugabyteDB from the [`sample` directory of the YugabyteDB GitHub repository](https://github.com/yugabyte/yugabyte-db/tree/master/sample). Download the following three files.
+The Chinook database that is compatible with YugabyteDB resides in the [`sample` directory of the YugabyteDB GitHub repository](https://github.com/yugabyte/yugabyte-db/tree/master/sample). The following three files will be used for this exercise. 
 
 - [`chinook_ddl.sql`](https://raw.githubusercontent.com/yugabyte/yugabyte-db/42799a519726c75f502f463795ac6cd3ebda40c2/sample/chinook_ddl.sql) — Creates the tables and constraints
 - [`chinook_genres_artists_albums.sql`](https://raw.githubusercontent.com/yugabyte/yugabyte-db/42799a519726c75f502f463795ac6cd3ebda40c2/sample/chinook_genres_artists_albums.sql) — Loads artist and album information
 - [`chinook_songs.sql`](https://raw.githubusercontent.com/yugabyte/yugabyte-db/42799a519726c75f502f463795ac6cd3ebda40c2/sample/chinook_songs.sql) — Loads individual song information
 
+Optionally, you can download them , but it is not necessary since they are all present under the `share` folder of the Yugabyte root directory.
```

Comment:
```suggestion
Optionally, you can download them, but it is not necessary since they are all present under the `share` folder of the Yugabyte root directory.
```